.github/actions/e2e/install-karpenter/action.yaml

```diff
@@ -83,7 +83,10 @@ runs:
         --set "serviceMonitor.endpointConfig.relabelings[2].replacement=$(git describe --abbrev=0 --tags)" \
         --set "serviceMonitor.endpointConfig.relabelings[3].targetLabel=commitsAfterTag" \
         --set "serviceMonitor.endpointConfig.relabelings[3].replacement=\"$(git describe --tags | cut -d '-' -f 2)\"" \
-        --wait
+
+      if [[ "$K8S_VERSION" == '1.23' ]]; then 
```

Comment:
Comment on why we are adding this annotation, just for context going forward. I don't expect us to have to keep this around for very long since we are about to drop support for 1.23 but would just be good to mark it down for now.